SPECS/libtar/libtar-1.2.11-mem-deref.patch

```diff
@@ -0,0 +1,24 @@
+--- libtar-1.2.11/lib/libtar.h.deref	2009-12-30 16:37:03.790121122 +0100
```

Comment:
NIT: The actual upstream patch for this seems to be: `560911b694055b0c677431cf85d4d0d5ebd1a3fd`

Can we please add a little context to this patch so that it becomes easy for us latter when we bump the version :)

How about something like `git format-patch -1 560911b694055b0c677431cf85d4d0d5ebd1a3fd`

Reference: https://repo.or.cz/libtar.git/patch/560911b694055b0c677431cf85d4d0d5ebd1a3fd

Reply:
Huh, interesting. Redhat is usually good about grabbing the actual formatted patches. I've updated to the upstream patch, 
thanks.